pkg/http/server.go

```diff
@@ -96,6 +99,21 @@ func (s *server) useCors(next fasthttp.RequestHandler) fasthttp.RequestHandler {
 	return corsHandler.CorsMiddleware(next)
 }
 
+func useAPIAuthentication(next fasthttp.RequestHandler) fasthttp.RequestHandler {
+	token := auth.GetAPIToken()
```

Comment:
Should we parse the token ( JWT ? ) in the beginning , add a check if the token is valid or not in the format ?

Reply:
No, Dapr doesn't care if it's a JWT token or not. The user is free to choose whatever API token they want.